cypress/e2e/internal/billing/scenarios/change-billing-acount-current-year.cy.js

```diff
@@ -0,0 +1,377 @@
+
+    // Test Region supplementary bill run
+    // we have to wait till the bill run has finished generating. The thing we wait on is the READY label. Once that
+    // is present we can check the rest of the details before confirming the bill run
+    cy.get('#main-content > div:nth-child(1) > div > p > strong', { timeout: 20000 }).should('contain.text', 'Ready')
+    cy.get('#main-content > div:nth-child(2) > div > h2').should('contain.text', '£582.11')
+    cy.get('#main-content > div:nth-child(4) > div > h2').should('contain.text', '4 supplementary bills')
+    cy.get('.govuk-button').contains('Confirm bill run').click()
+
+    // You're about to send this bill run
+    // check the details then click Send bill run
+    cy.get('dl').within(() => {
+      // date created
+      cy.get('@formattedCurrentDate').then((formattedCurrentDate) => {
+        cy.get('div:nth-child(1) > dd').should('contain.text', formattedCurrentDate)
+      })
+      // region
+      cy.get('div:nth-child(2) > dd').should('contain.text', 'Test Region')
+      // bill run type
+      cy.get('div:nth-child(3) > dd').should('contain.text', 'Supplementary')
+      // status
+      cy.get('div:nth-child(4) > dd').should('contain.text', 'Ready')
+    })
+    cy.get('.govuk-button').contains('Send bill run').click()
+
+    // Test Region Supplementary bill run
+    // spinner page displayed whilst the bill run is 'sending'. Confirm we're on it
+    cy.get('#main-content > div:nth-child(2) > div > p.govuk-body > strong').should('contain.text', 'Sending')
+    cy.get('#main-content > div:nth-child(2) > div > p.govuk-body-l')
+      .should('contain.text', 'The bill run is being created. This may take a few minutes.')
+    cy.get('#main-content > div:nth-child(7) > div > p')
+      .should('contain.text', 'Gathering transactions for old charge scheme')
+
+    // Bill run sent
+    // confirm the bill run is sent and then click to go to it
+    cy.get('.govuk-panel__title', { timeout: 20000 }).should('contain.text', 'Bill run sent')
+    cy.get('#main-content > div > div > p:nth-child(4) > a').click()
+
+    // Test Region supplementary bill run
+    // confirm we see it is now SENT
+    cy.get('#main-content > div:nth-child(1) > div > p > strong').should('contain.text', 'Sent')
+
+    // click the Bill runs menu link
+    cy.get('#navbar-bill-runs').contains('Bill runs').click()
+
+    // Bill runs
+    // back on the bill runs page confirm our PRESROC bill run is present and listed as SENT
+    cy.get('@formattedCurrentDate').then((formattedCurrentDate) => {
+      cy.get('#main-content > div:nth-child(5) > div > table > tbody > tr:nth-child(1)')
+        .should('contain.text', formattedCurrentDate)
+        .and('contain.text', 'Old charge scheme')
+        .and('contain.text', 'Test Region')
+        .and('contain.text', 'Supplementary')
+        .and('contain.text', '£582.11')
+        .and('contain.text', 'Sent')
+    })
```

Comment:
This is another section where we are waiting on the PRESROC bill run to complete generating, then we're sending it and confirming it's been sent. All this can be scrapped and we can go straight to the SROC bill run in this scenario.